source/MaterialXRuntime/RtFileIo.cpp

```diff
@@ -487,7 +492,17 @@ namespace
                         if (!socket)
                         {
                             const RtIdentifier inputType(elem->getType());
-                            RtInput input = nodegraph.createInput(socketName, inputType);
+
```

Comment:
Just a suggestion, but you could convert this into one or two lines if you wanted to:

const uint32_t flags = elem->isA<Token>() ? RtPortFlag::UNIFORM | RtPortFlag::TOKEN : 0;
RtInput input = nodegraph.createInput(socketName, inputType, flags);

or even:
RtInput input = nodegraph.createInput(socketName, inputType, elem->isA<Token>() ? RtPortFlag::UNIFORM | RtPortFlag::TOKEN : 0);

Reply:
I can do the first. Bit too hard to read the second :).